manifests/osfamily/aix.pp

```diff
@@ -51,11 +51,11 @@
   if $puppet_agent::absolute_source {
     $source = $puppet_agent::absolute_source
   } elsif $puppet_agent::alternate_pe_source {
-    $source = "${puppet_agent::alternate_pe_source}/packages/${pe_server_version}/aix-${aix_ver_number}-power/${puppet_agent::package_name}-${puppet_agent::prepare::package_version}-1.aix${aix_ver_number}.ppc.rpm"
+    $source = "${puppet_agent::alternate_pe_source}/packages/${pe_server_version}/aix-power/${puppet_agent::package_name}-${puppet_agent::prepare::package_version}-1.aix${aix_ver_number}.ppc.rpm"
```

Comment:
Isn't that a breaking change? For which PE or for which AIX releases is that required? It would be helpful if you could make PE-37302 public or move it to a public project.

Reply:
Yes, I was moving it back to draft PR.

Reply:
In order to preserve backwards compatibility, I believe we need to branch on the PE version. If the version is less than whatever the next version will be (either 2023.6.0 or 2021.7.7), then we need to include the `aix-${aix_ver_number}-ppc` in the URL. Otherwise, if it's equal or greater than, then use the generic `aix-ppc` This applies to `$puppet_agent::alternate_pe_source`, `$puppet_agent::source` and the `else` branches.

Side note, it